### ISSUE TITLE
ZMI: Use monospace font for "size" and "last modified"

### ISSUE THREAD

**Issue by dataflake** [zopefoundation/Zope#531]
In the ZMI folder listing, the "Size" and "Last Modified" columns look jumbled and are hard to parse visually because the font is not monospaced.

Please change the font to be monospaced like it used to be in Zope 2.

**Comment by drfho** [comment]
hi @dataflake, both Zope2/4 do not use monospaced fonts in the object list.  Despite that I can hardly see a jumbling!? To me it looks perfect. Can you provide a screen photo, please? 
Thanks fh

![zmi_date](https://user-images.githubusercontent.com/29705216/55243387-6a384280-523f-11e9-9a0f-bc77f1a0bd71.gif)

**Comment by dataflake** [comment]
The Verdana used in Zope 2 is not a true monospaced font, that is true, but it "acts" like one for the numbers displayed, see the nice aligned display in Zope 2 below. This is on macOS on Firefox, Chromium and Vivaldi, btw.

Zope 2:
![zope2](https://user-images.githubusercontent.com/1215784/55244413-24fc1c80-520f-11e9-8ea5-239846c4a2bb.png)

Zope 4:
![zope4](https://user-images.githubusercontent.com/1215784/55244437-2f1e1b00-520f-11e9-988d-f109cf637f2f.png)

**Comment by dataflake** [comment]
P.S.: Here's what the inspector says. Is this a macOS font picking issue?

![Screen Shot 2019-03-29 at 10 47 07](https://user-images.githubusercontent.com/1215784/55244924-1c581600-5210-11e9-98ae-49472f47b21b.png)

**Comment by drfho** [comment]
@dataflake 
wow, that picture actually is surprising. But for me it looks more like a problem with the font-size or family and not of the font-widths (number chars usually are "monospaced"). Please try to remove the first font-family declarations for the body element; maybe the Apple system fonts occur in variable size!? If this prevents the jumbling,  we should add a corresponding CSS overwrite to the top of:
Zope\src\zmi\styles\resources\zmi_base.css
```
/* ZOPE 4 Base Styles */
body.zmi {
font-family: "Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol"; 
}
```

**Comment by dataflake** [comment]
The real culprit here is `-apple-system`. Your fix works, thank you , please see and review PR #532
